api/rpc/perms/permissions.go

```diff
@@ -0,0 +1,36 @@
+package perms
```

Comment:
Weird again that it's not moved. Were there any changes to it?

Reply:
I didn't just rename the file, i also added in `ReadPerms`